### DISCUSSION TITLE
[RFC] More props of Chatbox

### DISCUSSION THREAD

**Discussion by MadCcc** [ant-design/ant-design#48510] [RFCs]
## Slots

Ref: https://github.com/ant-design/ant-design/discussions/48374

Decorations for Chatbox is also necessary in biz, for example:
![image](https://github.com/ant-design/ant-design/assets/27722486/0f35e6e2-4e76-444a-977f-914f96e698af)

### API

| Props | Description | Type | Default |
| --- | --- | --- | --- |
| header | Top slot of chatbox | ReactNode | - |
| footer | Bottom slot of chatbox | ReactNode | - |
| extra | Side slot of chatbox, the opposite side of avatar | ReactNode | - |

## Progress 

![image](https://github.com/ant-design/ant-design/assets/27722486/208b00b0-cf5f-4586-b91a-d62f281e4956)

### API

| Props | Description | Type | Default |
| --- | --- | --- | --- |
| progress | Progress bar at the bottom of chatbox | `{ percent: number }` | - |

**Comment by afc163** [comment]
不要叫 customXxx。
